docs/installation-aks.md

```diff
@@ -0,0 +1,407 @@
+
+This creates an identity for Open Service Broker for Azure to use when provisioning
+resources on your account on behalf of Kubernetes.
+
+1. Create a service principal with RBAC enabled:
+    ```console
+    az ad sp create-for-rbac --name osba -o table
+    ```
+1. Save the values from the command output in environment variables:
+
+    **Bash**
+    ```console
+    export AZURE_TENANT_ID=<Tenant>
+    export AZURE_CLIENT_ID=<AppId>
+    export AZURE_CLIENT_SECRET=<Password>
+    ```
+
+    **PowerShell**
+    ```console
+    $env:AZURE_TENANT_ID = "<Tenant>"
+    $env:AZURE_CLIENT_ID = "<AppId>"
+    $env:AZURE_CLIENT_SECRET = "<Password>"
+    ```
+
+## Configure the cluster with Open Service Broker for Azure
+
+You can now deploy Open Service Broker for Azure on the cluster. Using Helm:
+
+**Bash**
+
+```console
+helm repo add azure https://kubernetescharts.blob.core.windows.net/azure
+helm install azure/open-service-broker-azure --name osba --namespace osba \
+  --set azure.subscriptionId=$AZURE_SUBSCRIPTION_ID \
+  --set azure.tenantId=$AZURE_TENANT_ID \
+  --set azure.clientId=$AZURE_CLIENT_ID \
+  --set azure.clientSecret=$AZURE_CLIENT_SECRET \
+  --set redis.embedded=false \
+  --set redis.host=$REDIS_HOSTNAME \
+  --set redis.redisPassword=$REDIS_PASSWORD
+```
+
+**PowerShell**
+
+```console
+helm repo add azure https://kubernetescharts.blob.core.windows.net/azure
+helm install azure/open-service-broker-azure --name osba --namespace osba `
+  --set azure.subscriptionId=$env:AZURE_SUBSCRIPTION_ID `
+  --set azure.tenantId=$env:AZURE_TENANT_ID `
+  --set azure.clientId=$env:AZURE_CLIENT_ID `
+  --set azure.clientSecret=$env:AZURE_CLIENT_SECRET `
+  --set redis.embedded=false `
+  --set redis.host=$env:REDIS_HOSTNAME `
+  --set redis.redisPassword=$env:REDIS_PASSWORD
+```
+
+## Backup and Recovery
+
+### etcd
+
+The etcd cluster created above should have automatic backup enabled to Azure Blob Storage. The `etcd-backup` CronJob created by the Helm chart will periodically create new `etcdbackup` custom resources. These will not be removed if you remove the Helm chart.
+
+To initiate a restore of the etcd cluster, you need to create an instance of the `EtcdRestore` Custom Resource. A template for this can be found in contrib/k8s/etcd-operator/restore-operation.yaml
+
+```yaml
+apiVersion: etcd.database.coreos.com/v1beta2
+kind: EtcdRestore
+metadata:
+  name: svc-cat-etcd-restore
+spec:
+  etcdCluster:
+    name: svc-cat-etcd
+  backupStorageType: ABS
+  abs:
+    path: <abs continer>/<BACKUP-FILE> 
+    absSecret: <abs-credential-secret-name> 
+```
+
+You will need to replace the spec.abs.path value with the backup you'd like to restore from. You will need the storage container as well as the file name. For example, if your absSecret was called `dandy-clownfish-svc-cat-etcd` and you used etcd-backups as the storage container as directed above and the file name was etcd.backup.2018-06-12_19:31:05, your restore yaml would look like:
```

Comment:
let's add some backticks around `spec.abs.path` because ... codeblocks, all the beautiful codeblocks.